package.json

```diff
@@ -8,7 +8,8 @@
     "build": "next build",
     "start": "next start",
     "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "test": "eslint . --ext=tsx,ts"
```

Comment:
can you create a separate script for this to be just `lint` and `test` would be `lint` and soon to be jest tests too :)

Reply:
Done!